client/blocks/comments/form.jsx

```diff
@@ -19,7 +19,7 @@ import { getCurrentUser } from 'state/current-user/selectors';
 import { writeComment, deleteComment, replyComment } from 'state/comments/actions';
 import { recordAction, recordGaEvent, recordTrackForPost } from 'reader/stats';
 import { isCommentableDiscoverPost } from 'blocks/comments/helper';
-import { ProtectFormGuard } from 'lib/protect-form';
+import { ProtectFormGuard } from 'lib/protect-form/index.tsx';
```

Comment:
I wish I understood why this change was necessary and how to prevent it.

Reply:
It's the server build that crashes here:
```
ERROR in ./client/blocks/comments/form.jsx
Module not found: Error: Can't resolve 'lib/protect-form' in '/Users/jsnajdr/src/wp-calypso/client/blocks/comments'
 @ ./client/blocks/comments/form.jsx 36:19-46
 @ ./client/blocks/comments/form-root.jsx
 @ ./client/blocks/conversations/list.jsx
 @ ./client/blocks/reader-post-card/conversation-post.jsx
 @ ./client/blocks/reader-post-card/index.jsx
 @ ./client/components/seo/reader-preview/index.js
 @ ./client/components/seo-preview-pane/index.jsx
 @ ./client/components/web-preview/content.jsx
 @ ./client/components/web-preview/component.jsx
 @ ./client/components/web-preview/index.jsx
 @ ./client/my-sites/themes/theme-preview.jsx
 @ ./client/my-sites/theme/main.jsx
 @ ./client/my-sites/theme/controller.jsx
 @ ./client/my-sites/theme/index.node.js
 @ ./client/sections.js
 @ ./server/pages/index.js
 @ ./server/boot/index.js
 @ ./index.js
```
You need to add the TypeScript extensions also to the `webpack.config.node.js`.

Reply:
ahhhhh!!!! thanks!!!